server/webapp/WEB-INF/applicationContext-acegi-security.xml

```diff
@@ -260,6 +260,8 @@
                 /admin/materials/**=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
                 /admin/package_repositories/**=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
                 /admin/package_definitions/**=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
+                /api/admin/pipelines=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
+                /api/admin/pipelines/*=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
```

Comment:
@jyotisingh — I don't understand this DSL very much, so I'm not sure what the role would be in these cases —

```
GET .../pipelines - list pipelines, requires ROLE_USER (for future, we don't have a list pipelines call yet)
POST .../pipelines - create pipelines, requires ROLE_GROUP_SUPERVISOR, if the group already exists. requires ROLE_SUPERVISOR if the group does not exist
DELETE|POST|PUT .../pipelines/foo - requires atleast ROLE_GROUP_SUPERVISOR
```